### DISCUSSION TITLE
✨[feature Request]:  whole template Change

### DISCUSSION THREAD

**Discussion by kekubhai** [Anjaliavv51/Retro#508] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

I feel the theme of  retro doesn't go with the template we are using here, I suggest We should collaborate and try something like this 
![Retro   Old Computer Aesthetics](https://github.com/user-attachments/assets/20cf82f1-811d-4a0f-a8a4-a189af5ba721)
. I will try to Make some of the changes please assign  this to me. 

### Use Case

This will give the real retro vibe to the website and it might stand out

### Add ScreenShots

![Uploading Retro & Old Computer Aesthetics.jpeg…]()

### Priority

Medium

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24EXTD contributor
- [X] I have starred the repository

[a comment by an automated account is not shown]

**Comment by Anjaliavv51** [comment]
@kekubhai  go ahead.

**Comment by kekubhai** [comment]
Creating the entire website in that theme is very toiling for me alone, I just changed the navbar for now, Please check it and merge it if good enough. Thank you

**Reply by Anjaliavv51** [reply]
yes
